src/widgets/ScholiaWidget.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="scholia">
+    <div v-if="!lines">No commentary found.</div>
+    <div v-for="line in lines" :key="line.idx" class="line">
+      <span class="lemma">{{ line.lemma }} </span>
+      <span class="comment">{{ line.comment }}</span>
+    </div>
+  </div>
+</template>
+
+<script>
+  import gql from 'graphql-tag';
+  import { URN } from '@scaife-viewer/scaife-widgets';
+  import { MODULE_NS } from '@/reader/constants';
+
+  export default {
+    scaifeConfig: {
+      displayName: 'Scholia',
+      location: 'sidebar',
+      singleton: true,
+    },
+    computed: {
+      urn() {
+        return this.$route.query.urn
+          ? new URN(this.$route.query.urn)
+          : this.$store.getters[`${MODULE_NS}/firstPassageUrn`];
+      },
+      lines() {
```

Comment:
Not a blocker or anything to change, but just noting that the `textAnnotations` connection could return things other than scholia...it models any kind of annotation we want to attach to a line of text.

They also are better thought of as "annotations" than "lines", there might be multiple annotations per "line" (or whatever the lowest text part is in the passage), or a single annotation might refer to multiple lines in the passage.

I think there might need to be different widgets (or possibly more integration with the reader itself) for those other annotations.

![](https://github.trello.services/images/mini-trello-icon.png)  [Generalize text annotations widget](https://trello.com/c/MViOqOq7)